### DISCUSSION TITLE
Syntax highlight for Svelte

### DISCUSSION THREAD

**Discussion by OlegGulevskyy** [LazyVim/LazyVim#1164] [Q&A] [answered]
Hi!
I have successfully installed Svelte LSP and get all the intelliesense working well, however I am not sure how to get syntax highlighting as well for it? 
This is what I get (you can see hover works well with LSP)
<img width="615" alt="image" src="https://github.com/LazyVim/LazyVim/assets/43781031/8a23d6ea-993b-4895-bd65-f753f5399d77">

Would be forever grateful for any guidance!

**Comment by amaanq** [comment, marked as answer]
`:TSInstall svelte` or add it to tree-sitter's `ensure_installed`

**Comment by OlegGulevskyy** [comment]
Thanks a lot! This did the trick

**Reply by JayeshVP24** [reply]
Hey @OlegGulevskyy 
I'm having trouble setting Svelte's LSP.
Can u give a short path on how you did it?
I'm using LazyVim as well.

**Reply by OlegGulevskyy** [reply]
Hey @JayeshVP24 
I don't remember exactly, but if my memory doesn't fail:
1. Created a separate file `/plugins` directory (as Lua files get imported automatically from this folder) called treesitter and put this into it:
```lua
return {
  {
    "nvim-treesitter/nvim-treesitter",
    opts = {
      ensure_installed = {
         ...
        "svelte",  < --- make sure you have svelte LSP, the rest is your treesitter plugins
      },
    },
  },
}
```

And then install the LSP through the terminal as well, I think it was done with "TSInstall svelte", but I remember I had a picker to install different LSPs but I can't remember how I triggered it :D
I am not a pro neovim user, so might mix up the termins and this probably doesn't help you much, but hope it does give you an inspiration

**Reply by JayeshVP24** [reply]
Thanks @OlegGulevskyy 
My problem was LSP not updating typescript types after a change in file saved to disk. Found out, its a problem with Neovim 0.9.1.
Updated to 0.10 nightly version and all's well again. 
I asked u coz u mentioned your intellisense was working well.
Thanks again!

**Comment by pandasoli** [comment]
@OlegGulevskyy Sorry for being out of topic, but what's the intelliesense plugin you're using?
Pretty minimalist, very cool

**Reply by OlegGulevskyy** [reply]
> @OlegGulevskyy Sorry for being out of topic, but what's the intelliesense plugin you're using? Pretty minimalist, very cool

It's been a year and I have changed since then but I think it's a Github theme
https://github.com/projekt0n/github-nvim-theme

P.S. if you are referring to the one I provide in the screenshot, then that is the actual problem - theme is not working there, because syntax highlighting was not working properly.

**Reply by pandasoli** [reply]
I'm referring about the theme, but the auto complete plugin you use, like nvim-cmp, coc...
